x-pack/plugins/cases/server/authorization/types.ts

```diff
@@ -90,20 +90,30 @@ export interface OperationDetails {
   savedObjectType: string;
 }
 
+/**
+ * Describes an entity with the necessary fields to identify if the user is authorized to interact with the saved object
+ * returned from some find query.
+ */
+export interface OwnerEntity {
+  owner: string;
+  id: string;
+}
+
+/**
+ * Function callback for making sure the found saved objects are of the authorized owner
+ */
+export type EnsureSOAuthCallback = (entities: OwnerEntity[]) => void;
+
 /**
  * Defines the helper methods and necessary information for authorizing the find API's request.
  */
-export interface AuthorizationFilter {
+export interface AuthFilterHelpers {
   /**
    * The owner filter to pass to the saved object client's find operation that is scoped to the authorized owners
    */
   filter?: KueryNode;
   /**
    * Utility function for checking that the returned entities are in fact authorized for the user making the request
    */
-  ensureSavedObjectIsAuthorized: (owner: string) => void;
-  /**
-   * Logs a successful audit message for the request
-   */
-  logSuccessfulAuthorization: () => void;
+  ensureSavedObjectsAreAuthorized: EnsureSOAuthCallback;
```

Comment:
Just wanted to call this out: The Authorization class is no longer returning `logSuccessfulAuthorization` since that produced a duplicate message for read operations (these helpers are used for read operations).